content/blog/set-up-load-balancer-with-metallb-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Setting up the load balancer with MetalLB in HPE GreenLake for Private
+  Cloud Enterprise
+date: 2023-09-01T08:59:11.280Z
+author: Guoping Jia
+authorimage: /img/guoping.png
+disable: false
+tags:
+  - Kubernetes
+  - MetalLB
+  - Load Balancer
+  - HPE GreenLake
+  - HPE GreenLake for Private Cloud Enterprise
+  - HPE GreenLake for Containers
+---
+This blog post describes how to set up the load balancer using [MetalLB](https://metallb.universe.tf/) for a Kubernetes (K8s) cluster in [HPE GreenLake for Private Cloud Enterprise](https://www.hpe.com/us/en/greenlake/private-cloud-enterprise.html). It allows customers to configure load balancing services for their workloads deployed in K8s clusters. 
+
+### Overview
+
+Using [HPE GreenLake for Private Cloud Enterprise: Containers](https://www.hpe.com/us/en/greenlake/containers.html), customers can create services with the type of *NodePort* for their workloads deployed in K8s clusters using the label *hpecp.hpe.com/hpecp-internal-gateway=true*. The services will be automatically exposed to a container platform gateway host with assigned ports. The deployed workloads will become accessible externally using the gateway host name and the assigned ports as access URLs. 
+
+Different from various public cloud providers, such as *GCP*, *AWS* and *Microsoft Azure*, HPE GreenLake for Private Cloud Enterprise doesn’t support network load balancers by default. As cluster administrators, after you create K8s cluster, you can integrate cluster with any load balancers in place to support K8s services of type *LoadBalancer*. This blog post will show you how to use MetalLB to provide load balancing services for K8s clusters in HPE GreenLake for Private Cloud Enterprise. It provides customers with the flexibility to configure custom load balancers for their deployed workloads in HPE GreenLake for Private Cloud Enterprise.
+
+### Prerequisites
+
+Before starting, make sure you meet the following requirements:
```

Comment:
If you add the following, the bullet list will use the same styl as the rest of the text

<style>
li {
   font-size: 27px;
   line-height: 33px;
   max-width: none;
}
</style>

Reply:
Thank you for pointing me this. I've added this, but updated the values to font-size / line-height:

<style> li { font-size: 100%; line-height: 23px; max-width: none; } </style>

Otherwise, the font / line look too big. Not sure if I got it correctly.

Reply:
If it's ok with you, it'ok for me too :-)

Reply:
I like this markdown css. I will definitely reuse it in my coming blog post :-)